modules/services/twmn.nix

```
@@ -0,0 +1,347 @@
+# twmn service
+{ config, lib, pkgs, stdenv, ... }:
+with lib;
+let
+  cfg = config.services.twmn;
+
+  animationOpts = {
+    curve = mkOption {
+      type = types.ints.between 0 40;
+      default = 38;
+      example = 19;
+      description = ''
+        the qt easing-curve animation to use for the animation, See
+        <link href="https://doc.qt.io/qt-5/qeasingcurve.html#Type-enum">
+        QEasingCurve documentation</link>.
+      '';
+    };
+    duration = mkOption {
```

Please insert an empty line between option definitions, otherwise it becomes so dense and difficult to navigate.
```suggestion

    duration = mkOption {
```